### PR TITLE
Fix highlighting cols/rows headers when using non-contiguous selection

### PR DESCRIPTION
**Pull request by budnix**
### Context

These changes are required to support header selection to work with the non-contiguous selection.

### How has this been tested?

It was tested manually by selecting multiple non-contiguous header selections.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Related issue(s):
1. #4875 

### Checklist:

- [x] My code follows the code style of this project,
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by budnix** [comment]
Related PRO PR https://github.com/handsontable/handsontable-pro/pull/50

File: test/helpers/asciiTable.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  const hasHighlight = cell.classList.contains('ht__highlight');
+
+  let symbol = '   ';
+
+  if (hasActiveHeader) {
+    symbol = ' * ';
+
+  } else if (hasHighlight) {
+    symbol = ' - ';
+
```
Comment (swistach) [inline test/helpers/asciiTable.js]: Could you remove this unnecessary line?

File: test/helpers/asciiTable.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+  const consumedFlags = new Map([
+    ['hasLeftHeader', hasLeftHeader],
+    ['hasTopHeader', hasTopHeader],
+    ['hasCornerHeader', hasCornerHeader],
+    ['hasFixedLeftCells', hasFixedLeftCells],
+    ['hasFixedTopCells', hasLeftHeader],
+  ]);
+
+  for (let r = 0; r < masterTable.rows.length; r++) {
```
Comment (swistach) [inline test/helpers/asciiTable.js]: Could you add cache for 'masterTable.rows.length'?

**Comment by budnix** [comment]
Added required changes.

**Comment by budnix** [comment]
@swistach Can you recheck this PR?

**Pull request merged**